vscode-lean4/src/leanclient.ts

```diff
@@ -127,9 +128,22 @@ export class LeanClient implements Disposable {
                 env
             }
         }
+
+        const documentSelector: DocumentFilter = {
+            language: 'lean4'
+        }
+
+        if (this.workspaceFolder){
+            documentSelector.scheme = 'file'
+            documentSelector.pattern = `${this.workspaceFolder.uri.toString()}/**/*`
```

Comment:
BTW, this was the wrong pattern. :cry: 

https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/VS.20Code.200.2E0.2E60.20-.20Manual.20refresh.20necessary

Reply:
Interesting.  Weird that the toString() version seemed to be working fine on my machine.   On Windows your change to 'fsPath' produces a weird windows path of "d:\temp\lean_examples\foo/**/*" which seemed wrong to me, but I guess it still works.  Thanks for fixing it.